aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -237,6 +237,16 @@ module std::features {
         is_enabled(MODULE_EVENT)
     }
 
+    /// Whether the aggregator snapshots feature is enabled.
+    /// Lifetime: transient
+    const AGGREGATOR_SNAPSHOTS: u64 = 29;
```

Comment:
I'm just curious how these numbers are determined, to make sure we aren't missing some convention (or I'm not missing some convention) - is it just anything?

Reply:
There is a Features enum containing all the feature flags, where each feature flag is assigned number sequential numbers. I just used the next number available in that enum.